### ISSUE TITLE
Change requested for AppCenter: MPRIS data reported incorrectly

### ISSUE THREAD

**Issue by cassidyjames** [naaando/lyrics#5]
It seems really finicky with MPRIS, often reporting the incorrect data (such as no player found, or not playing when it is, etc.). Due to this I could never seem to get it to display lyrics.

**Comment by naaando** [comment]
Thank you for the feedback, my app is a lyric viewer but requires a python program in order to provide the lyrics and for some reasons I wanted to keep them separated for now, so did you installed the [lyricsources](https://github.com/naaando/lyricsources) complement in order to provide automatic lyrics download? I would like to submit those packages separated to AppCenter requiring the provider package on Debian control of my app, is it possible?

**Comment by cassidyjames** [comment]
Apps may only depend on standard packages that are available in elementary OS. It is not possible to upload your own non-GUI package separately and have your app depend on that. Instead, you could include the package in your app, either via a git submodule or by including the source directly in the tree.

**Comment by naaando** [comment]
Alright, I'm shipping it as a git module now.

**Comment by cassidyjames** [comment]
@naaando I'm not sure this is being installed correctly still in the latest release. Using [this generated deb](http://packages.elementary.io/testing/pool/main/c/com.github.naaando.lyrics/com.github.naaando.lyrics0.1.7-df6348.deb) on Juno, I still get:

```bash
com.github.naaando.lyrics 

** (com.github.naaando.lyrics:27423): WARNING **: 10:08:21.419: LyricSourcesRepository.vala:17: Error calling StartServiceByName for org.lyricsources.LyricSourcePlugin.viewlyrics: GDBus.Error:org.freedesktop.DBus.Error.Spawn.ChildExited: Process org.lyricsources.LyricSourcePlugin.viewlyrics exited with status 1

(com.github.naaando.lyrics:27423): GLib-GObject-CRITICAL **: 10:08:21.623: Flags for property 'state' on class 'MprisPlayer' remove functionality compared with the property on interface 'LyricsPlayer'


(com.github.naaando.lyrics:27423): GLib-GObject-CRITICAL **: 10:08:21.623: Flags for property 'busname' on class 'MprisPlayer' remove functionality compared with the property on interface 'LyricsPlayer'


(com.github.naaando.lyrics:27423): GLib-CRITICAL **: 10:08:21.627: g_variant_get_int64: assertion 'g_variant_is_of_type (value, G_VARIANT_TYPE_INT64)' failed
** Message: 10:08:21.628: LocalRepository.vala:31: looking for lyric file: 11 Save Rock And Roll.lrc

** (com.github.naaando.lyrics:27423): WARNING **: 10:08:21.661: LyricSourcesRepository.vala:17: Error calling StartServiceByName for org.lyricsources.LyricSourcePlugin.viewlyrics: GDBus.Error:org.freedesktop.DBus.Error.Spawn.ChildExited: Process org.lyricsources.LyricSourcePlugin.viewlyrics exited with status 1

** (com.github.naaando.lyrics:27423): CRITICAL **: 10:08:21.661: lyric_sources_downloader_search: assertion 'self != NULL' failed

(com.github.naaando.lyrics:27423): GLib-GObject-WARNING **: 10:08:21.661: invalid (NULL) pointer instance

(com.github.naaando.lyrics:27423): GLib-GObject-CRITICAL **: 10:08:21.661: g_signal_connect_data: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed
```

**Comment by naaando** [comment]
Alright, I'll test on a VM to understand what's going on in a clean environment.

**Comment by naaando** [comment]
I think it should work now, submitted a new release for review on AppCenter, if everything runs well I close this issue.

**Comment by cassidyjames** [comment]
This particular issue appears to be fixed in the 0.2.4 release!

**Comment by naaando** [comment]
Cool! It was a problem with python dependencies that was preventing the lyrics provider from start, but this error should be better handled too.
